easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.2.0-GCC-12.3.0.eb

```diff
@@ -0,0 +1,69 @@
+easyblock = 'Bundle'
+
+name = 'SentencePiece'
+version = '0.2.0'
+
+homepage = 'https://github.com/google/sentencepiece'
+description = "Unsupervised text tokenizer for Neural Network-based text generation."
+github_account = 'google'
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('pkgconf', '1.9.5'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('gperftools', '2.12'),
+    ('protobuf', '24.0'),
+    ('Abseil', '20230125.3'),
+]
+
+default_component_specs = {
+    'source_urls': [GITHUB_LOWER_SOURCE],
+    'sources': ['v%(version)s.tar.gz'],
+    'checksums': ['9970f0a0afee1648890293321665e5b2efa04eaec9f1671fcf8048f456f5bb86'],
+}
+
+local_external_absl = 'sed -i %(builddir)s/'
+
+components = [
+    (name, version, {
+        'easyblock': 'CMakeMake',
+        'separate_build_dir': True,
+        'start_dir': '%(namelower)s-%(version)s',
+        'configopts': '-DSPM_PROTOBUF_PROVIDER=package -DSPM_ABSL_PROVIDER=package',
+    }),
+    ('sentencepiece', version, {
+        'easyblock': 'PythonPackage',
+        'start_dir': '%(namelower)s-%(version)s/python',
+        # Unpredicable where pc files end up; including both lib and lib64
+        'prebuildopts': 'export PKG_CONFIG_PATH=%(installdir)s/lib64/pkgconfig:$PKG_CONFIG_PATH && ',
+        'preinstallopts': 'export PKG_CONFIG_PATH=%(installdir)s/lib64/pkgconfig:$PKG_CONFIG_PATH && ',
```

Comment:
Until https://github.com/easybuilders/easybuild-framework/pull/4435 (i.e. 5.0) you will need to extend PKG_CONFIG_PATH with both lib64 and lib directory versions